demo/sdl2surface_rawfb/main.c

```diff
@@ -128,20 +130,22 @@ grid_demo(struct nk_context *ctx)
 
 int main(int argc, char **argv)
 {
+    //Define some varibles for later use
     struct nk_color clear = {0,100,0,255};
     struct nk_vec2 vec;
     struct nk_rect bounds = {40,40,0,0};
     struct sdlsurface_context *context;
 
+    NK_UNUSED(argc);
+    NK_UNUSED(argv);
```

Comment:
I believe these appear below the variable declarations to abide by C89 needing dlvariable declaration first.